packages/components/src/templates/teaser-media.stories.ts

```diff
@@ -0,0 +1,106 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Teaser Media',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ *
+ * - Aligment: Teaser contents can be center aligned if desired.
+ * - Paddings: Can be changed as desired.
+ * - Headline size: Can be changed as desired.
+ *
+ * ```
+ * ```
+ */
+
+export const Default = {
+  render: () =>
+    html` <style>
+        #part-content sd-teaser-media::part(content) {
+          padding: 60px;
+        }
+      </style>
+      <div class="flex flex-wrap gap-8 w-[600px]">
+        <div id="part-content">
```

Comment:
There's no need to wrap the teaser in this case, you can add the `part-content` as an id or class directly in the component and style it from there.

Reply:
Updated